bindata/v4.1.0/aws-pod-identity-webhook/deployment.yaml

```diff
@@ -26,6 +26,7 @@ spec:
             drop: [ "ALL" ]
         command:
         - /usr/bin/aws-pod-identity-webhook
+        - --aws-default-region=${REGION}
```

Comment:
Set this to a sane default, which will be used when infra resource does not specify a region.
```
- --aws-default-region=us-east-1
```

Reply:
cool, updated!
add added an unit test case for this fallback scenario.